SmartDeviceLink-iOS/SmartDeviceLink/SDLProtocol.m

```diff
@@ -28,6 +28,7 @@ @interface SDLProtocol () {
     dispatch_queue_t _sendQueue;
     SDLPrioritizedObjectCollection *_prioritizedCollection;
     NSMutableDictionary *_sessionIDs;
+    NSMutableDictionary *_hashIDs;
```

Comment:
Either dynamic enough or private enough to change it without consequence. Probably the latter.

I'm considering the change in this security branch to track which services are encrypted, so that if you try to send an encrypted RPC on a non-encrypted service type it fails, I think this would fit in the same place. So I would say yeah, it probably fine for now, but should change sooner rather than later. Maybe as a "hotfix" separate from this.